apcd-cms/src/apps/view_users/templates/view_users.html

```diff
@@ -14,22 +14,35 @@ <h1>View Users</h1>
   <hr />
   <p style="margin-bottom: 30px">View submitted users.</p>
   <hr />
-  <div class="filter-container">
+  <div class="filter-container" style="display: flex; gap: 10px;">
```

Comment:
```suggestion
  <div class="filter-container">
```
Related to `filter-content` comment, becomes unneeded.